src/enums/componentsCompanyInfo.ts

```diff
@@ -21,5 +20,6 @@ export enum ComponentsCompanyInfo {
   'association-type',
   'create-special-resolution',
   'approval-type-section',
+  'relationships-panel',
```

Comment:
I also fixed this error:

![image](https://github.com/bcgov/business-edit-ui/assets/10002889/d508a419-be45-443f-8200-edc4cca063c6)

See this object's comments (lines 2-5) for rules!